BaseFilter/sections/foreign.txt

```diff
@@ -10650,11 +10650,8 @@ phimsexmoi.biz###text-2[style*="left: 0px; top: 0px;"]
 nhaccuatui.com###eventBannerLeft
 nhaccuatui.com###eventBannerTop
 nhaccuatui.com##div[class^="adv_"]
-truyenqqhot.com###ad_info_top
-truyenqqhot.com###ad_info
-truyenqqhot.com###ads_info
-truyenqqhot.com###ad_infos
-truyenqqhot.com##.text_ads
+truyenqqhot.com##div[id^="ad_"]:not([id*="_poster"])
+truyenqqhot.com##div[id^="ads_"]
```

Comment:
I don't see where the `truyenqqhot.com##div[id^="ads_"]` rule are needed?

Reply:
> I don't see where the `truyenqqhot.com##div[id^="ads_"]` rule are needed?

they have ad ids called `ads_top` sometimes `ad_top` looks like they will randomly update these id names

Reply:
> I don't see where the `truyenqqhot.com##div[id^="ads_"]` rule are needed?

anyways updating the filter when they change to `ads_top` is very time consuming so i think its better to include in this pull request